src/test/prediction_service_utils_test.cpp

```diff
@@ -78,31 +78,31 @@ class ModelWithModelInstanceFakeLoad : public ovms::Model {
 
 std::shared_ptr<ModelWithModelInstanceFakeLoad> modelWithModelInstanceFakeLoad;
 
-TEST_F(GetModelInstanceTest, WithRequested0VersionUnloadedShouldReturnModelNotLoadedAnymore) {
+TEST_F(GetModelInstanceTest, WithRequestedDefaultVersionUnloadedShouldReturnNoModelVersionAvailable) {
```

Comment:
Test specifies that NO_MODEL_VERSION_AVAILABLE will be returned but it checks if status is MODEL_VERSION_MISSING.